pkg/controller/backup_storage_location_controller.go

```diff
@@ -59,8 +60,8 @@ func (r *BackupStorageLocationReconciler) Reconcile(ctx context.Context, req ctr
 
 	locationList, err := storage.ListBackupStorageLocations(r.Ctx, r.Client, req.Namespace)
 	if err != nil {
-		log.WithError(err).Error("No backup storage locations found, at least one is required")
-		return ctrl.Result{}, err
+		return ctrl.Result{},
+			fmt.Errorf("error: %s, no backup storage locations found, at least one is required", err.Error())
```

Comment:
Let's make sure the stack trace is returned with the error here.

Reply:
Got it. Updated according to comments.